setup.py

```diff
@@ -63,9 +63,6 @@
         "Operating System :: POSIX :: Linux",
         "Topic :: System :: Systems Administration",
         "Topic :: Utilities",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
```

Comment:
Should I add Python 3.10?

Reply:
I'm not sure to be honest. If this is a declaration of what we promise to be compatible with, at this point that's just 3.9. But the purpose of this metadata could be broader than that.